src/PrizePool.sol

```diff
@@ -484,14 +499,29 @@ contract PrizePool is Manageable, Multicall {
             claimCount++;
         }
         claimRecords[_winner] = ClaimRecord({drawId: lastCompletedDrawId, claimedTiers: uint8(BitLib.flipBit(claimRecord.claimedTiers, _tier))});
+        UD60x18 deltaExchangeRate = _computeTierDeltaExchangeRate(prizeSize, _tier);
+        // console2.log("deltaExchangeRate", deltaExchangeRate.unwrap());
+        // console2.log("_tierExchangeRates[_tier]", _tierExchangeRates[_tier].unwrap());
+        UD60x18 test = ud(_tierExchangeRates[_tier].unwrap() + deltaExchangeRate.unwrap());
+        _tierExchangeRates[_tier] = _tierExchangeRates[_tier].add(test);
+        // console2.log("GOT HERE????");
         prizeToken.transfer(_to, payout);
         if (_fee > 0) {
             claimerRewards[_feeRecipient] += _fee;
         }
         emit ClaimedPrize(lastCompletedDrawId, _vault, _winner, _tier, uint152(payout), _to, _fee, _feeRecipient);
+        // console2.log("ALLL DONE");
         return prizeSize;
     }
 
+    function _computeTierDeltaExchangeRate(uint256 _prizeSize, uint8 _tier) internal view returns (UD60x18) {
+        if (_tierExchangeRates[_tier].gte(prizeTokenPerShare)) {
+            return ud(0);
+        }
+        UD60x18 deltaExchangeRate = prizeTokenPerShare.sub(_tierExchangeRates[_tier]);
+        return deltaExchangeRate.mul(toUD60x18(_prizeSize)).div(toUD60x18(_getLiquidity(_tier, tierShares)));
```

Comment:
Inline this equation and add a comment above to explain the calculation.
```suggestion
        return prizeTokenPerShare.sub(_tierExchangeRates[_tier]).mul(toUD60x18(_prizeSize)).div(toUD60x18(_getLiquidity(_tier, tierShares)));
```

Reply:
`_tierExchangeRates[_tier]` is called in this function but also in `_getLiquidity`. Same for other functions calling `_getLiquidity`, so it would make sense to pass it directly to `_getLiquidity`.